firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/CrashlyticsWorker.java

```diff
@@ -60,6 +65,7 @@ public ExecutorService getExecutor() {
    * <p>Returns a <code>Task</code> which will be resolved upon successful completion of the
    * callable, or throws an <code>ExecutionException</code> if the callable throws an exception.
    */
+  @CanIgnoreReturnValue
```

Comment:
Can we add the comment `Submit a blocking callable will block the execution` as discussed?

Reply:
Oops forgot that, added!